src/tools/ddsperf/ddsperf.c

```diff
@@ -1939,11 +1963,12 @@ int main (int argc, char *argv[])
   char netload_if[256];
   double netload_bw = -1;
   double rss_init = 0.0, rss_final = 0.0;
+  char filename[256] = { 0 };
```

Comment:
There's no need to copy the file name to the stack, one can just store `optarg`.

Reply:
Used optarg to open the file.